examples/tutorials/core_api_pytorch_mnist/model_def_checkpoints.py

```diff
@@ -97,11 +100,23 @@ def test(args, model, device, test_loader, epoch, core_context, steps_completed)
     )
 
 
-# NEW: Define load_state function for restarting model training from existing checkpoint.
-def load_state(checkpoint_directory):
+# NEW: Define load_state function for restarting model training from existing checkpoint. Returns (.pt, int).
+# Also update load_state header to take trial info object as an argument.
+def load_state(checkpoint_directory, info):
     checkpoint_directory = pathlib.Path(checkpoint_directory)
-    with checkpoint_directory.joinpath("checkpoint.pt").open("rb") as f:
-        return torch.load(f)
+
+    with checkpoint_directory.joinpath("state").open("r") as f:
+        epochs_completed, ckpt_trial_id = [int(field) for field in f.read().split(",")]
+
+    # NEW: If the saved trial ID is the same as our current trial ID, we are resuming the trial, so we should return the model weights
```

Comment:
isn't this whole function already marked as `# NEW:`?  I think you could omit the `NEW:` portion of this comment if you'd like.